src/styles/variables.css

```diff
@@ -1,65 +1,106 @@
   --primary: rgb(104, 210, 210); /* curreny colony primary */
-  --secondary: rgb(0, 230, 196); /* greenish primary color */
   --colony-white: rgb(251, 251, 251);
   --colony-black: rgb(31, 35, 39);
   --colony-blue: rgb(79, 93, 255);
-  --colony-light-blue: rgb(238, 242, 245);
   --dark-blue: rgb(69, 85, 230);
   --sky-blue: rgb(90, 145, 255);
   --danger: rgb(248, 43, 101); /* user mentions color */
-  --violet: rgb(89, 100, 240);
   --pink: rgb(254, 94, 124);
   --golden: rgba(255, 176, 0); /* Think pending states (golden yellow) */
   --dark: rgb(60, 68, 77); /* Used for colony name display (colony simplified) */
-  --grey-dark: rgb(47, 47, 47);
-  --grey-1: rgb(58, 58, 58);
-  --grey-2: rgb(54, 54, 54);
-  --grey-3: rgb(82, 84, 85);
-  --grey-4: rgb(124, 124, 124);
-  --grey-5: rgb(173, 186, 189);
-  --grey-blue-0: rgb(232, 232, 232);
-  --grey-blue-1: rgb(232, 236, 245);
-  --grey-blue-2: rgb(244, 246, 250);
-  --grey-blue-3: rgb(246, 246, 246);
-  --action-secondary: rgb(76, 93, 255);
+  --text-disabled: rgb(194, 204, 204);
+  --colony-light-blue: rgb(238, 242, 245);
+  --light-grey-blue: rgb(236, 238, 242);
+  --drop-shadow: rgba(62, 118, 244, 0.14);
 
-  /*
-   * We really need to cleanup the greys, as they're growing larger and larger by the day
-   * Maybe get a count of all the colors from product and try to go from there ...
-   */
-  --temp-grey-0: rgb(195, 204, 205);
-  --temp-grey-1: rgb(188, 197, 216);
-  --temp-grey-2: rgb(46, 50, 53);
-  --temp-grey-4: rgb(153, 153, 153);
-  --temp-grey-5: rgb(213, 213, 213);
-  --temp-grey-6: rgb(220, 226, 227);
-  --temp-grey-8: rgb(52, 55, 56);
-  --temp-grey-11: rgb(136, 136, 136); /* Used for the text color of time for the 'CardActivity' items */
-  --temp-grey-12: rgb(152, 164, 193); /* Used for Drag & Drop placeholder */
+  /* Remane `--grey-1` to `--dark-grey` */
+  --dark-grey: rgb(58, 58, 58);
+
+  /* Remane `--grey-4` to `--grey` */
+  --grey: rgb(124, 124, 124);
+
+  /* Remane `--grey-blue-3` to `--grey-navbar`, potentially to be removed as used only for navbar */
+  --grey-navbar: rgb(246, 246, 246);
+
+  /* Remane `--temp-grey-blue-5` to `--light-grey-pink` */
+  --light-grey-pink: rgb(244, 238, 242); /* Used for user profile gradient start color */
+
+  /* Remane `--temp-grey-blue-6` to `--light-ash` */
+  --light-ash: rgb(231, 242, 246); /* Used for user profile gradient end color */
+
+  /* Remane `--temp-grey-blue-4` to `--light-purple` */
+  --light-purple: rgb(200, 214, 247); /* Used for some box shadows */
+
+  /* Remane `--temp-grey-blue-7` to `--grey-purple` */
+  --grey-purple: rgb(118, 116, 139); /* Used for the wallet connect button in the user navigation */
+
+  /* Remane `--temp-grey-blue-1` to `--grey-blue` */
+  --grey-blue: rgb(188, 197, 216);
+
+  /* Remane `--temp-grey-5` to `--light-grey` */
+  --light-grey: rgb(213, 213, 213);
+
+  /* Remane `--temp-grey-13` to `--light-grey-beige` */
   --temp-grey-13: rgb(227, 224, 227); /* Used for separator background */
```

Comment:
Are we still keeping this "temp" grey ?

Reply:
No, it's `--light-grey-beige` now, somehow was skipped in my commit - good catch!